xml/System.Security.Cryptography/DSAOpenSsl.xml

```diff
@@ -80,9 +114,25 @@
         <Parameter Name="handle" Type="System.IntPtr" />
       </Parameters>
       <Docs>
-        <param name="handle">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="handle">The OpenSSL <c>DSA*</c> value to use as the key.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Security.Cryptography.DSAOpenSsl" /> class from an existing OpenSSL key represented as a <c>DSA*</c>.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+> [!IMPORTANT]
+> OpenSSL supports multiple library versions being loaded within the same process.
+> Before calling this constructor, verify your pointer value came from the same version of OpenSSL that this class uses. 
+> For more information, see <xref:System.Security.Cryptography.SafeEvpPKeyHandle.OpenSslVersion%2A>.
+
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentException">
+          <paramref name="handle" /> is <see cref="F:System.IntPtr.Zero"/>.</exception>
+        <exception cref="T:System.Security.Cryptography.CryptographicException">
+          <paramref name="handle"/> is not a valid <c>DSA*</c>.</exception>
+        <altmember cref="P:System.Security.Cryptography.SafeEvpPKeyValue.OpenSslVersion" />
```

Comment:
build warning due to malformed DocId.

The SafeEvpPKeyValue class should've been SafeEvpPKeyHandle.